docs/submission/submitting-clinical-data.md

```diff
@@ -63,3 +63,48 @@ Uploaded data must be checked for accuracy against the data in the current submi
 1. If your data does not require approval by DCC, it will be submitted automatically.
 
 Once submitted, you will see the clinical data reflected on your [Program Dashboard](/docs/submission/submitted-data).
+
+## Submitting data in Surgery file
```

Comment:
Suggestion, capitalize to be consistent with other L2 headings: "_Submitting Data in Surgery File_"

Reply:
@hknahal `File` still in lowercase (sorry to be picky!)

Reply:
Good catch @b-f-chan! I fixed this now: https://deploy-preview-512--argo-docs.netlify.app/docs/submission/submitting-clinical-data#submitting-data-in-surgery-file

Reply:
Looks good resolving!